api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/health/ApiCatalogHealthIndicator.java

```diff
@@ -22,19 +24,40 @@
  */
 @Component
 @RequiredArgsConstructor
+@Slf4j
 public class ApiCatalogHealthIndicator extends AbstractHealthIndicator {
+    private static String AUTHENTICATION_SERVICE_ID = "apiml.authorizationService.zosmfServiceId";
+    private static String AUTHENTICATION_SERVICE_PROVIDER = "apiml.authorizationService.provider";
+    private static String ZOSMF = "zosmf";
 
     private final DiscoveryClient discoveryClient;
 
     @Override
     protected void doHealthCheck(Health.Builder builder) {
         String gatewayServiceId = CoreService.GATEWAY.getServiceId();
+        String authServiceId = ZOSMF;
+        String authServiceProvider = ZOSMF;
+        if (!this.discoveryClient.getInstances(gatewayServiceId).isEmpty()){
+            authServiceId = this.discoveryClient.getInstances(gatewayServiceId).get(0).getMetadata().get(AUTHENTICATION_SERVICE_ID);
+            authServiceProvider = this.discoveryClient.getInstances(gatewayServiceId).get(0).getMetadata().get(AUTHENTICATION_SERVICE_PROVIDER);
+        }
+        boolean authServiceUp = false;
+        try {
+            if (authServiceProvider.equalsIgnoreCase(ZOSMF)) {
+                authServiceUp = !this.discoveryClient.getInstances(authServiceId).isEmpty();
+            } else if (authServiceProvider.equalsIgnoreCase("dummy")) {
+                authServiceUp = true;
+            }
+        } catch (AuthenticationServiceException ex) {
+            log.debug("The parameter 'zosmfServiceId' is not configured.", ex);
```

Comment:
Consider maybe using   the already existing message 
`- key: org.zowe.apiml.security.zosmfInstanceNotFound
      number: ZWEAG108` with apimlLog.log

Reply:
This is actually not needed anymore, it's a leftover and it does not throw the exception anymore. I will change this.